src/iop/cacorrect.c

```diff
@@ -440,7 +443,7 @@ void process(
       for(int top = -border; top < height; top += ts - border2)
```

Comment:
{

Nothing to do with your changes, but I'm surprised Pascal didn't ask for it :-)

Reply:
indeed, makes that huge loop better to review

Reply:
I hadn't reviews yet :)